django_spanner/__init__.py

```diff
@@ -31,7 +31,7 @@
 
 
 USING_DJANGO_3 = False
-if django.VERSION[:2] == (3, 2):
+if django.VERSION[:2] == (3, 2) or django.VERSION[:2] == (4, 2):
     USING_DJANGO_3 = True
```

Comment:
maybe just check if django version is >=3.2 and <= 5.2, and thrown an exception otherwise?
this will drop the support of 2.2

Reply:
also  USING_DJANGO_3 is semantically wrong once 4.2 is supported.